README.md

```diff
@@ -574,6 +575,28 @@ Type: `Boolean` Default: `false`
 
 Set to `true` to remove unused translation keys from i18n resource files.
 
+#### filterUnusedKeys
+
+Type: `Object` Default: `false`
+
+If an `Object` is supplied and `removeUnusedKeys` is set to `true`, to leave specific keys you can specify a function like so:
```

Comment:
Here is the refined doc:

====

Type: `Function` or `null` Default: `null`

This option allows you to define a custom function to filter unused keys. If a function is provided and `removeUnusedKeys` is set to true, the function will be used to specify which keys should be kept.